### DISCUSSION TITLE
Three new ways Revolt will be improving user safety - Revolt

### DISCUSSION THREAD

**Discussion by giscus** [stoatchat/discussions#503] [Announcements]
# Three new ways Revolt will be improving user safety - Revolt

Introducing in-app reporting, updated guidelines for Revolt Discover and built-in anti-spam measures.

https://revolt.chat/posts/improving-user-safety

**Comment by DeclanChidlow** [comment]
Wicked job on the new system!
I'm glad to hear that you are putting in the time to improve the platform in such a way. With any luck this should curb the remaining harassment accounts that have proved such a pain. Thanks for keeping the platform going!

**Comment by ashleyirispuppy143** [comment]
In-App Reporting is literally 1984 /s

**Reply by sussycatgirl** [reply]
your account has been banned and your IP address has been reported to the authorities

regards, revolt

**Comment by AkumuNyaaaa** [comment]
As my old discord user in 2016, I haven't seen this software, so I worth to join this?

**Comment by wolixoriginal** [comment]
this be amazing update for revolt
